CIP-0031/README.md

```diff
@@ -2,10 +2,13 @@
 CIP: 31
 Title: Reference inputs
 Authors: Michael Peyton Jones <michael.peyton-jones@iohk.io>
-Comments-Summary: No comments
-Comments-URI:
+Implementors: 
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+    - Jared Corduan <jared.corduan@iohk.io>
 Status: Active
-Type: Standards Track
+Categories: 
+    - Plutus
+    - Ledger
```

Comment:
The duplicate category seems perfectly understandable to me but this syntax still awaits confirmation via issue https://github.com/cardano-foundation/CIPs/issues/433 and probably also (@KtorZ) an update to https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#categories.

Reply:
There's actually no `Ledger` category.

Reply:
I mean, it still is not registered. We've added it for some existing CIPs; but I would refrain to add it to any new ones..

Reply:
I'm assuming there will be one, though.

Reply:
I've been assuming for a couple of months too now 🙃

Reply:
just noting for cross-reference: there is soon going to be one 🙂 https://github.com/cardano-foundation/CIPs/pull/456/files